features/cli/oc_import_image.feature

```diff
@@ -147,7 +147,7 @@ Feature: oc import-image related feature
       | resource        | dc                 |
       | o               | yaml               |
     Then the output should match:
-      | [Ll]astTriggeredImage.*:.*<%= project.name %>\/deployment-example@sha256.*|
+      | <%= project.name %>\/deployment-example@sha256 |
```

Comment:
The `/` is escaped there, but not in line 117.  Perhaps one of them will fail. Could you add test log ?

Reply:
No failure about it, the purpose is to test this difference.

Reply:
If they are the same, then it should be a bug.

Reply:
Pass log is added to the description.

Reply:
Ack